galley/testdata/validation/dataset.go

```diff
@@ -1,4 +1,4 @@
- // Copyright 2019 Istio Authors
+// Copyright 2019 Istio Authors
```

Comment:
We had a check, at one point, which validated the headers.  Is that check not running?  Can you take a look - seems like I've seen a load of PRs with this small change in the headers.  Might as well make them all at once.

Reply:
It seems like it may not be running. I already merged a PR with this fix in so I think it just shows up here since I haven't rebased